app/controllers/external_users/advocates/interim_claims_controller.rb

```diff
@@ -2,4 +2,9 @@ class ExternalUsers::Advocates::InterimClaimsController < ExternalUsers::ClaimsC
   skip_load_and_authorize_resource
 
   resource_klass Claim::AdvocateInterimClaim
+
+  def build_nested_resources
```

Comment:
I'm not a big fan of this approach of ensuring the associated resources are initialised for the purposes of form rendering, but for now I've decided to keep it similarly to the other controllers for consistency.